sampler.go

```diff
@@ -326,8 +326,8 @@ func newAdaptiveSampler(strategies *sampling.PerOperationSamplingStrategies, max
 	samplers := make(map[string]*GuaranteedThroughputProbabilisticSampler)
 	for _, strategy := range strategies.PerOperationStrategies {
 		samplers[strategy.Operation] = newGuaranteedThroughputProbabilisticSampler(
-			strategies.DefaultLowerBoundTracesPerSecond,
 			strategy.ProbabilisticSampling.SamplingRate,
+			strategies.DefaultLowerBoundTracesPerSecond,
 			defaultMaxSamplesPerSecond, // TODO (wjang) get the maxSamplesPerSecond from strategy
```

Comment:
I'll rebase this when https://github.com/uber/jaeger-client-go/pull/143 lands and then make the changes in this diff